index.html

```diff
@@ -5969,8 +5969,8 @@ <h4 id="elements_of_block_heading">
 <span its-locale-filter-list="en" lang="en">Elements of Block Heading</span>
 <span its-locale-filter-list="ja" lang="ja">別行見出しの構成</span>
 </h4>
-      <p its-locale-filter-list="en" lang="en">JIS X 4051 describes the elements of a block heading as follows: top level heading, medium level heading and low level heading have to have a label name, number, heading title and heading sub-title (see [[[#fig3_1_6]]]). Note that the label name, number and heading sub-title are not mandatory. </p>
-      <p its-locale-filter-list="ja" lang="ja">別行見出しの構成については，JIS X 4051では大見出し，中見出し及び小見出しについて，それぞれ<span class="index" id="d8e584">ラベル名</span>，<span class="index" id="d8e587">番号</span>，<span class="index" id="d8e590">見出し文字列</span>及び<span class="index" id="d8e593">副題</span>で構成するとしている（[[[#fig3_1_6]]]）．ただし，ラベル名，番号及び副題は，必須要素ではなく，省略してもよいとなっている．</p>
+      <p its-locale-filter-list="en" lang="en">JIS X 4051 describes the elements of a <a href="#term.block-heading" class="termref">block heading</a> as three levels: <a href="#term.top-level-heading" class="termref">top level heading</a>, medium level heading, and low level heading. These block headings SHALL have a heading title, and MAY have a <a href="#term.label-name" class="termref">label name</a>, number, and heading <a href="#term.subtitle" class="termref">subtitle</a> (see [[[#fig3_1_6]]]).</p>
```

Comment:
We don't use all uppercase letters for the RFC 2119 words in this document. I think we should follow the existing convention?

Reply:
updated, how about new one?

Reply:
Comparing with "have to", "require" sounds better to me. @r12a, what do you think?

Reply:
I don't think we need to worry about RFC 2119 for this document. Firstly, because it isn't a formal spec - it's all informative text. Secondly, because it would be a lot of work to apply it systematically and correctly to the whole document. Thirdly, because that convention doesn't have an equivalent that can be used in the Japanese version, and the two language versions should be the same.

Reply:
I agree with Richard. What do you suggest in this case? Does the current text in the PR look good to you?

Reply:
I think 'for three levels' may be better than 'as three levels'.

Reply:
'heading title' is referred to as 'heading text' in the diagrams (which sounds better to me, unless i'm missing something)

i'd be inclined to make 'label name' become 'label' in english

'heading subtitle' seems odd to me - maybe just 'subtitle' is sufficient (and is what is used for the diagram)

Reply:
fixed as 'for three levels', 'heading text', and (heading) 'subtitle' by ef64949.
For 'heading title', 'heading text' is used for any other place, and this is only one in whole document.
'label name' is also in glossary.
For 'heading subtitle', it is used only in this section, and just 'subtitle' is used in glossary.

Reply:
I'll ask to fix https://github.com/w3c/jlreq/pull/121 on 'heading' title/subtitle also.